app.rb

```diff
@@ -0,0 +1,74 @@
+require "sinatra/reloader"
+require "sinatra"
+require "active_record"
+
+# look for a gem so I Can use that in this file
+require_relative "models/apartment"
+require_relative "models/tenant"
+require_relative "db/connection"
+# look for a file nearby
+
+# address = []
+
+get "/apartments" do
+  @apartments = Apartment.all
+  erb :"apartments/index"
+end
+
+# create apartments
+post '/apartments' do
+  @apartment = Apartment.create(address: params[:address], monthly_rent: params[:monthly_rent], sqft: params[:sqft], num_beds: params[:num_beds], num_baths: params[:num_baths])
+  erb :"/apartment/new"
+  redirect "/apartments/#{@apartment.id}"
+end
+
+# delete apartments
+delete '/apartments/:address' do
+  @apartments = Apartment.find(params[:address])
+  @apartments.destroy
+  redirect("/apartments")
+end
+
+# edit apartments
+get "/apartments/:address/edit" do
+  @apartments = Apartment.find(params[:address])
+  erb :"apartmentss/edit"
+end
+
+# update apartements
+put '/apartments/:address' do
```

Comment:
Usually we'll use ID here, since that's the simplest unique thing that identifies an apartment. Also, addresses have spaces and other characters in that aren't fit for use in a URL.